documentation/specs/src/interoperability/ethereum-bridge/bootstrapping.md

```diff
@@ -3,8 +3,12 @@
 ## Overview
 
 The Ethereum bridge is not enabled at the launch of a Namada chain. Instead,
-there is a governance parameter, `eth_bridge_proxy_address`, which is
-initialized to the zero Ethereum address
+there are two governance parameters:
+
+- `eth_bridge_proxy_address`
+- `eth_bridge_wnam_address`
```

Comment:
I think any contracts that will not be upgradeable we could store as governance parameters. So perhaps the _Vault_ contract as well. Or we could store only the _Proxy_ address and have validators look up via their oracle the addresses of the other contracts as required. I'm not sure which is the better approach.

Reply:
I think we will allow Vault to be upgradeable, but hopefully that won't ever be necessary

Reply:
:+1: for this current spec I'll leave it as not specified to be upgradeable, we will need to consider an upgrade process for it if we want it